rfcs/text/0008/threat.yml

```diff
@@ -70,8 +88,14 @@
     type: keyword
     short: Indicator confidence rating
     description: >
-      Identifies the confidence rating assigned by the provider using STIX confidence
-      scales (N/H/M/L, 0-10, Admirality, WEP, or DNI).
+      Identifies the confidence rating assigned by the provider using STIX confidence scales.
```

Comment:
Is this field intended to be the raw, unnormalized value based on whichever confidence scale (0-10, WEP, DNI, etc.) the provider generating the indicator uses?

Will the normalized STIX confidence value (0-100) get dropped into a different field?

Reply:
Good question.

This is the raw, unnormalized value.